lib/formtools/utils.js

```diff
@@ -35,7 +35,7 @@ exports.getDefaultValue = function (field) {
             return value = field.defaultValue();
         }
 
-    } else if (field.defaultValue === undefined && fieldType === 'datetime') {
+    } else if (field.defaultValue === undefined) {
```

Comment:
Do you think this is appropriate without a check against the `fieldType`? It means if there is no default value, it will set the value to a date? That doesn't seem quite right to me?

Reply:
@allanchau, this one is still to be considered.

Reply:
@smebberson Should this block be removed altogether? So the default is null if not type is specified.

Reply:
@allanchau, that's what I'm thinking... As long as the default fields that are applied by Linz include a default, I think it should be fine. Do you agree?

Reply:
Yeah, sounds good to me.